set.mm

```diff
@@ -227529,15 +227698,6 @@ operation is a permutation group (group consisting of permutations), see
       HVBVDVFVG $.
   $}
 
-  ${
-    $d A f $.
-    $( The set of permutations of a set ` A ` exists.  (Contributed by AV,
-       30-Mar-2024.) $)
-    permsetex $p |- ( A e. V -> { f | f : A -1-1-onto-> A } e. _V ) $=
-      ( wcel cv wf1o cab wf cvv mapex anidms wss f1of ss2abi a1i ssexd ) ACDZAA
-      BEZFZBGZAARHZBGZIQUBIDAACCBJKTUBLQSUABAARMNOP $.
-  $}
-
```

Comment:
Do not remove this or the next proof, at least not until we evaluate the difference in proofs and axiom usage, for instance, replacement.

Reply:
OK, I will mark them as OLD.

Reply:
I think they are not obsoleted by the new theorems, since they do not use ax-rep (while the others do?), and their proofs are more direct. Or maybe "ALT".

It will be easier to judge when we see the html pages. It seems wiser to wait and see if we can then make them obsolete, rather than first obsoleting them and then possibly having to backtrack..

Reply:
After I revised ~mapfoss, ~f1osetex does not depend on ~ax-rep anymore. Moreover, ~f1osetex  and ~permsetex depend on the same axioms now:

  ax-mp ax-1 ax-2 ax-3 ax-gen ax-4 ax-5 ax-6 ax-7 ax-8 ax-9 ax-10 ax-11 ax-12
  ax-ext ax-sep ax-nul ax-pow ax-pr ax-un

As for the proof, I do not see any added value in the proof of ~permsetex: it is primarily based on ~mapex using ~f1of, whereas the proof of ~f1osetex is based on ~mapfoss (via ~fosetex) using ~f1ofo. Is it "more direct" to go back from bijections to arbitrary functions instead of surjections? 

~ symgbasex can be (and was) replaced by using ~eqeltri and ~fvex, which together use much less axioms. Therefore, I see no need in this theorem, too.

Reply:
Ok. We'll have a look at the generated html pages.